### PR TITLE
PR to review task related to CollectionType

### PR DESCRIPTION
**Pull request by sajalMittal2004**
This PR is to get reviewed the Task given related to Apex collection type.

### CODE REVIEW

File: ApexCollectionType.cls [hunk 1]
```diff
@@ -0,0 +1,75 @@
+public class ApexCollectionType {
+
```
Comment (Nikhil-Fexle-May2022) [inline ApexCollectionType.cls]: Remove unwanted spaces.

File: ApexCollectionType.cls [hunk 2]
```diff
@@ -0,0 +1,75 @@
+public class ApexCollectionType {
+    
+    public static void practiceOnMap(){
+        Map<String, List<String>> mapToStoreDepartmentAndRelatedEmployee = new Map<String, List<String>>();
+
```
Comment (Nikhil-Fexle-May2022) [inline ApexCollectionType.cls]: Remove unwanted spaces.

File: ApexCollectionType.cls [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        for(Integer i = 0; i < numberList.size(); i++){
+            for(Integer j = 0; j < numberList.size()-1-i; j++){
+                if(numberList[j] > numberList[j+1]){
+                    Integer temp = numberList[j];
+                    numberList[j] = numberList[j+1];
+                    numberlist[j+1] = temp;
+                }
+            }
+        }
+
```
Comment (Nikhil-Fexle-May2022) [inline ApexCollectionType.cls]: Do Proper code indentation.

File: ApexCollectionType.cls (hunk 2 above)
Comment (sajalMittal2004) [reply ApexCollectionType.cls]: Sir, can I add one line space to differentiate the operations(Union, intersection etc.) and to enhance the readability or remove them also?

Comment (Nikhil-Fexle-May2022) [reply ApexCollectionType.cls]: Do whatever you want, but the code and the class should be properly indented.

**Pull request closed without merging**